customersupport/static/list-tickets.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+    
+    $("#listTicketsButton").click(function(){
+        $.get("api/ticket/customer",{customer_id:$("#customer").val()})
+            .done(function (data) {
+                $("#listTicketSelection").find("option").remove();
+                $.each(data, function(index, item){
+                    $("#listTicketSelection").append(
+                        $('<option></option>')
+                            .text(`id: ${item.id} customer_id: ${item.customer_id}  date: ${item.date_opened} issue: ${item.issue} status: ${item.current_status}`)
```

Comment:
Did What I could for now, this is a good candidate for an enhancement for the future.